app/models/registration.rb

```diff
@@ -1,8 +1,8 @@
 # frozen_string_literal: true
 
 class Registration < ApplicationRecord
-  scope :pending, -> { where(accepted_at: nil, deleted_at: nil, is_competing: true) }
-  scope :accepted, -> { where.not(accepted_at: nil).where(deleted_at: nil) }
+  scope :pending, -> { where(competing_status: 'pending') }
```

Comment:
unintentional change because my DB doesn't have accepted_at and deleted_at anymore. Will remove if we merge this before the bye bye v1 PR

Reply:
Now byebye-v1 is merged, so this should be noop